cmd/juno/juno.go

```diff
@@ -106,6 +106,36 @@ func main() {
 		return nil
 	})
 
+	var location, network string
+
+	snapshotCommand := &cobra.Command{
```

Comment:
We should probably put this in a separate file `cmd/juno/snapshot.go`. @kirugan probably knows more about Cobra idioms, but that's what I've seen in other projects.

Reply:
Moving it outside the `main` function also allows us to test it.

Reply:
I don't see what can be tested since the presence of flag values is already handled by Viper and the required flags are already handled by Cobra. Do you suggest something else to test ?

Reply:
Ah yes I think you're right. Ideally we move as much stuff out of Cobra/Viper's control and rigorously test everything we do control. Apologies for the false-positive review comments, trying to review quickly xD